scripts/format.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# format the source files
+if [ "$#" -eq 0 ]
+then
+  # format source files
+  yapf -i -r setup.cfg securedrop bin
```

Comment:
I meant remove the entire `--style setup.cfg`. I think it can just be `yapf -i -r securedrop bin`

Reply:
Done